armory/baseline_models/keras/keras_gtsrb_micronnet.py

```diff
@@ -0,0 +1,68 @@
+"""
```

Comment:
During PR for Scenario subclass we should make this `baseline_models/keras/gtsrb_micronnet.py`